docs/scos/dev/front-end-development/202212.0/oryx/getting-started/oryx-environment.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Environment Variables"
+---
+
+Environment variables are useful when:
+
+- Values are different across developer machines.
+- Values are different across multiple environments: (dev, staging, qa, prod)
+- Values change frequently and are highly dynamic.
```

Comment:
```suggestion
- Values should not leak in the code repository
- Values are different across developer machines.
- Values are different across multiple environments: (dev, staging, qa, prod)
- Values change frequently and are highly dynamic.

```

Reply:
Added, also rephrased intro block a bit.

Reply:
I'd say the secret part is one of the bullets why you'd need an env var. Secrets like api keys cannot be in the code base, which is why you put them in an env var. I'd add this one as the 1st of your list.